src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="margin-form"
+      action="#"
```

Comment:
add correct action from task's repository

Reply:
where i can find it?